src/arduino-rfm/LoRaMAC.cpp

```diff
@@ -62,13 +63,14 @@
 *****************************************************************************************
 */
 void LORA_Cycle(sBuffer *Data_Tx, sBuffer *Data_Rx, RFM_command_t *RFM_Command, sLoRa_Session *Session_Data,
- 									sLoRa_OTAA *OTAA_Data, sLoRa_Message *Message_Rx, sSettings *LoRa_Settings)
+ 									sLoRa_OTAA *OTAA_Data, sLoRa_Message *Message_Rx, sSettings *LoRa_Settings, msg_t *upMsg_Type)
 {
+	Serial.println("cyclee..");

```

Comment:
This Serial print should not been here, or it should depend on a debug macro

Reply:
> I am sorry that I make changes a bit of carelessly since I dont have sufficent time

I forgot to delete this line of code :)